conduit/plugins/importers/syncmode/syncmode_importer_config.go

```diff
@@ -0,0 +1,9 @@
+package syncmode
+
+// Config specific to the sync mode importer
+type Config struct {
+	// Algod netaddr string
+	NetAddr string `yaml:"netaddr"`
+	// Algod rest endpoint token
+	Token string `yaml:"token"`
+}
```

Comment:
Can you add the docs generation? I've just copied this from the algod importer. The readme is in https://github.com/algorand/indexer/tree/develop/cmd/conduit-docs if you want to change or add anything.
```suggestion

//go:generate conduit-docs ../../../../conduit-docs/

//Name: conduit_importers_syncmode

// Config specific to the sync mode importer
type Config struct {
	// <code>netaddr</code> is the Algod network address. It must be either an <code>http</code> or <code>https</code> URL.
	NetAddr string `yaml:"netaddr"`
	// <code>token</code> is the Algod API endpoint token.
	Token string `yaml:"token"`
}
```